src/views/lesson.pug

```diff
@@ -84,28 +87,44 @@ block content
 			.mb-6(class="shadow-[0_2px_10px_0_rgba(0,0,0,0.1)]")
 				+homework(homework)
 
+	if timeStart
+		p#timeStart.hidden #{timeStart || 0}
+
 append scripts
+	script(src="/js/copyLink.js")
 	if loggedIn
 		script(src="/js/hwDone.js")
 		script(src="/js/hwProgress.js")
 		script(src="/js/lessonProgress.js")
-
 	if lesson.videoId && !lesson.twitchVideo
+		
 		script.
 			var tag = document.createElement('script');
 			tag.src = "https://www.youtube.com/iframe_api";
 			var firstScriptTag = document.getElementsByTagName('script')[0];
 			firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
 			var player;
+			const timeStart = document.getElementById('timeStart')?.innerText || 0;
```

Comment:
Janky: and i get the hidden value here.

Im open to a better solution but I couldn't resolve the variable scope issue. variable `timeStart` is not available here in the inline script section